deploy/eck-elasticsearch/Chart.yaml

```diff
@@ -3,7 +3,7 @@ name: eck-elasticsearch
 description: A Helm chart to deploy Elasticsearch managed by the ECK Operator.
 kubeVersion: ">= 1.21.0-0"
 type: application
-version: 0.3.0
+version: 0.4.0
```

Comment:
I'm not sure we should bump the version. We could put this change in `0.3.0` which has not been released yet?